lib/cdo/slog.rb

```diff
@@ -58,8 +58,9 @@ def key_for(host)
     def parse(buffer)
       buffer.split("\n").map do |entry|
         info, payload = entry.split(', ', 2)
```

Comment:
Note that `payload` is set `nil` here, if `entry` does not contain the substring ", ".